index.es.js

```diff
@@ -5,6 +5,7 @@ import { createFilter } from 'rollup-pluginutils'
 export default function css (options = {}) {
   const filter = createFilter(options.include || ['/**/*.css', '/**/*.scss', '/**/*.sass'], options.exclude)
   let dest = options.output
+  const runtime = options.runtime || require('node-sass')
```

Comment:
By moving the call to require up here, the plugin won't catch possible errors: If runtime is not given and node-sass is not installed (for example someone upgrading) this will throw an error (module not found).